### PR TITLE
Add Helper Text

### PR DESCRIPTION
**Pull request by Kirakishou**
Help text comes from help_strings.json and are sent as a command

### CODE REVIEW

File: help_strings.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+    "rules": [
+      "Type '!random' in chat and the bot will play ",
```
Comment (Brainicism) [inline help_strings.json]: Is there a reason this is an array of split clauses?

File: kmq.js [hunk 2]
```diff
@@ -95,6 +99,47 @@ client.on('message', message => {
     }
 });
 
+// Usage: `!help [action]` or `!help`
+const help = (message, action) => {
+    var embed_title = "";
```
Comment (Brainicism) [inline kmq.js]: Change all `var`s to `let`s

File: kmq.js [hunk 3]
```diff
@@ -95,6 +99,47 @@ client.on('message', message => {
...
+            embed_fields.push({
+                name: argument.name,
+                value: argument.description.join("")
+            })
+        });
+    }
+    else {
+        embed_title = "KMQ Command Help"
+        embed_desc = helpMessages.rules.join("")
+        helpMessages.actions.forEach(function(action) {
```
Comment (Brainicism) [inline kmq.js]: Change callback to arrow function

File: help_strings.json (hunk 1 above)
Comment (Kirakishou) [reply help_strings.json]: readability

Comment (Brainicism) [reply help_strings.json]: I think it'd make more logical sense to keep it as a single string. Text wrapping should be handled by the IDE

File: kmq.js [hunk 4]
```diff
@@ -70,6 +73,47 @@ client.on("message", (message) => {
     }
 });
 
+// Usage: `!help [action]` or `!help`
+const help = (message, action) => {
+    let embed_title = "";
```
Comment (Brainicism) [inline kmq.js]: Going forward, we should use camelCase for variables but I'll just fix this manually

**Pull request merged**